.github/CODEOWNERS

```diff
@@ -1,8 +1,6 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
-
 # More details are here: https://help.github.com/articles/about-codeowners/
-
 # The '*' pattern is global owners.
 
-* kminehart
+* @grafana/grafana-bi-squad
```

Comment:
Looks like we don't have write access in this repository. Can you please add us?

Reply:
I'm tracking it down now 😄

Reply:
We have it 🥳